uni/lib/controller/local_storage/preferences_controller.dart

```diff
@@ -162,8 +177,16 @@ class PreferencesController {
 
   /// Deletes the user's student number and password.
   static Future<void> removePersistentUserInfo() async {
-    await prefs.remove(_userNumber);
-    await prefs.remove(_userPw);
+    await _secureStorage.delete(key: _userNumber);
+    await _secureStorage.delete(key: _userPw);
+    await prefs.remove(_userFaculties);
+  }
+
+  /// Deletes the user's session refresh token.
+  static Future<void> removeSessionRefreshToken() async {
+    await _secureStorage.delete(key: _userNumber);
+    await _secureStorage.delete(key: _refreshToken);
+    await prefs.remove(_userFaculties);
```

Comment:
Why is this line needed?

Reply:
the _userFaculties? Because it also saving the faculties in preferences, as in the credentials authentication